hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/TestSCMCommonPlacementPolicy.java

```diff
@@ -286,6 +287,33 @@ public void testReplicasWithoutMisreplication() {
     Assertions.assertEquals(0, replicasToCopy.size());
   }
 
+  @Test
```

Comment:
While I have not reviewed the logic under test yet (waiting for the description of how it work as per my earlier comment), I suspect there are more test scenarios needed that just this one simple one?

Reply:
Have added a additional testcases.